picard/ui/mainwindow.py

```diff
@@ -386,6 +386,10 @@ def create_statusbar(self):
             "you \"Search\" or \"Open in Browser\" from Picard, clicking the "
             "\"Tagger\" button on the web page loads the release into Picard."
         ))
+        self.eta_label = QtWidgets.QLabel()
+        self.eta_label.setMaximumSize(QtWidgets.QLabel("ETA: 00d:00h:00m:00s").size())
```

Comment:
This should be initialized with actual format from a function so format is in one place (something like `eta_dhms(0)` returning translated string)

Reply:
In addition I even think this should be initialited in the actual implementation. See my comment on refactoring this below.